index.js

```diff
@@ -27,10 +27,15 @@ const HEADER = `// Type definitions for WebExtension Development in FireFox ${ar
 // TypeScript Version: 2.4
 // Generated using script at github.com/bomjacob/definitelytyped-firefox-webext-browser
 
-interface WebExtEventListener<T extends (...args: any[]) => any> {
-    addListener: (callback: T) => void;
-    removeListener: (listener: T) => void;
-    hasListener: (listener: T) => boolean;
+interface WebExtEventBase<TAddListener, TCallback> {
+    addListener: TAddListener;
+
+    removeListener (cb: TCallback): void;
+
```

Comment:
it looked better without empty lines

Reply:
I totally agree, my IDE derped on me :)

Reply:
Actually my IDE adds the newlines afterwards when i format the file anyways so *shrugs*